### PR TITLE
Adds the Weeping Stars, some Coalition regions,  new start sound effects

### PR DESCRIPTION
**Pull request by kyres1**
This PR will add the functioning Weeping Stars region as planned. It also adds new roundstart sound effects for the Badlands and the new region(s) added. This also adds a lot of specific documentation to the space_sectors.dm file to specify what should be seen where.

The Coalition menu buttons:
dreamseeker_NrUUyIoON2

An example of the skybox;
dreamseeker_duTua9GZgP

The new stargazers;
image

To-do,
- [x] Add whatever lore input is given on the cargo pricing
- [x] Figure out what third parties go where with lore

Sound effects were bashed together in Audacity by me using random bits and pieces from freesound.

### CODE REVIEW

**Comment by kyres1** [comment]
!wip

**Comment by Greenjoe12345** [comment]
image
the stargazer in the bridge hallway will want to be moved one tile downwards

**Comment by kyres1** [comment]
!review

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

File: maps/away/away_site/civ_station/civilian_station.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "civilian station"
 	description = "A modestly-sized independently-owned civilian space station. Many of these exist all throughout inhabited space, offering a place to rest, food to eat, shopping, and refueling - part mall, part motel. This one appears to have been active in the region long before Biesel took control, and an information lookup indicates that it is operated by a small company that is Solarian in origin. This one's transponder says it's open for business!"
 	suffixes = list("away_site/civ_station/civilian_station.dmm")
-	sectors = list(SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
+	sectors = list(SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, ALL_COALITION_SECTORS)
```
[the remark this answers is not in the document]
Comment (NonQueueingMatt) [reply maps/away/away_site/civ_station/civilian_station.dm:5]: the description implies these were built during the solarian hegemony, so it'd be fitting

**Pull request merged**